### PR TITLE
Ian edit create

### PR DESCRIPTION
**Pull request by LeeMellon**
Basic CRU(sans D) functionality.

### CODE REVIEW

File: src/app/components/articles/article-edit/article-edit.component.ts [hunk 1]
```diff
@@ -41,29 +42,42 @@ export class ArticleEditComponent implements OnInit {
...
-      } else {
-        // "res" should be null-or-undefined, maybe need different message?
-        alert('trouble editing the article' + res);
-      }
-    } catch (err) {
-      console.log(err);
-    }
+    // try {
+      const res = this.articleSvc.updateArticle(this.authInfo.uid, this.userInfo, article, this.key);
+    //   if (res) {
```
Comment (Methodician) [inline src/app/components/articles/article-edit/article-edit.component.ts]: This is the functionality we need back. Is it not working?

File: src/app/components/articles/article-post/article-post.component.html [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+<div class="postArticleContainer">
+    <div class="upload-wrapper">
+      <label>Main Image</label>
+      <div class="uploadForm">
+        <input accept="image/*" id="control" type="file" (change)="detectFiles($event)">
+      </div>
+    </div>
+    <cos-article-form #form>
+    </cos-article-form>
+    <button (click)="save(form.value)" mat-raised-button color="warn" [disabled]="!form.valid">Post</button>
```
Comment (Methodician) [inline src/app/components/articles/article-post/article-post.component.html]: The disabled binding doesn't seem to be working... or maybe it is and the form is just showing valid when it's not?

**Comment by LeeMellon** [comment]
Updated.

File: src/app/components/articles/article-edit/article-edit.component.ts (hunk 1 above)
Comment (LeeMellon) [reply src/app/components/articles/article-edit/article-edit.component.ts]: This is done.

**Pull request merged**